eng/Directory.Packages.props

```diff
@@ -192,6 +192,7 @@
     <PackageVersion Include="System.Runtime.CompilerServices.Unsafe" Version="6.0.0" />
     <PackageVersion Include="System.Runtime.Loader" Version="4.3.0" />
     <PackageVersion Include="System.Resources.Extensions" Version="7.0.0" />
+    <PackageVersion Include="System.Security.Permissions" Version="8.0.0" />
```

Comment:
I'm still concerned about this one. It has a dependency on System.Windows.Extensions. Is System.Security.Permissions even needed for Linux builds?

Reply:
See also this discussion: https://github.com/dotnet/msbuild/issues/9213

Reply:
This is what's nice about the transitive pinning feature: specifying this means that if any of our dependencies do pull this in, we'll get a version that won't cause problems. But if nobody uses it, this will have no effect.

Reply:
Got it, thanks.